addons/wiki/static/wikiPage.js

```diff
@@ -86,7 +86,13 @@ function ViewWidget(visible, version, viewText, rendered, contentURL, allowMathj
                 request.done(function (resp) {
                     if(self.visible()) {
                         var $markdownElement = $('#wikiViewRender');
-                        var rawContent = resp.wiki_content || '*Add important information, links, or images here to describe your project.*';
+                        if (resp.wiki_content){
+                            var rawContent = resp.wiki_content
+                        } else if(window.contextVars.currentUser.canEdit) {
```

Comment:
This is probably fine, but we generally avoid using `window.contextVars` in modules, except for `-page` modules.

You could use `lodashGet` to ensure that accessing `window.contextVars.currentUser.canEdit` won't error, even when any of the properties are unset.

Not a big deal, though.